firehose.c

```diff
@@ -568,7 +577,7 @@ static int firehose_program(struct qdl_device *qdl, struct program *program, int
 
 			vip_transfer_clear_status(qdl);
 		}
-		n = qdl_write(qdl, buf, chunk_size * sector_size, 30000);
+		n = qdl_write(qdl, buf, chunk_size * sector_size, zlp_timeout);
```

Comment:
The actual increase to 30s occurred in the previous commit `"qdl: Be explicit about storage types"`. Could you rephrase the commit message to more accurately reflect the changes made in that commit, or transfer that change (`s/10000/30000/g`) to this commit?